woocommerce/woocommerce-android
# [IPP] Track error type and error description in the human readable way

Closes: #8859


### Description

The PR make payment errors we track more readable in tracks

### Testing instructions

Look into logcat for `card_present_collect_payment_failed` tracking event
* Try to collect 1.01 payment
* Try to modify `PaymentManager::169` with something like `is CapturePaymentResponse.Successful -> emit(errorMapper.mapCapturePaymentError(paymentIntent, CapturePaymentResponse.Error.ServerError("server error message")))`

Notice that the error type and description are readable and clear now



```
Tracked: card_present_collect_payment_failed, Properties: {"payment_method_type":"card","country":"US","error_context":"CardReaderTracker","error_description":"Capturing payment failed: ServerError(errorMsg=server error message)","error_type":"Server(errorMessage=server error message)","card_reader_battery_level":"85 %","plugin_slug":"woocommerce-payments","currency":"USD","card_reader_model":"STRIPE_M2","blog_id":192152755,"is_wpcom_store":true,"is_debug":true}
```

### Images/gif
